tools/generator/src/Generator/SetTargetConfigurations.swift

```diff
@@ -209,6 +209,17 @@ Target with id "\(id)" not found in `consolidatedTarget.uniqueFiles`
             )
         }
 
+        let exportedSymbolsLists = target.inputs.exportedSymbolsLists
+        if !exportedSymbolsLists.isEmpty {
+            print(target.inputs.exportedSymbolsLists)
+            try buildSettings.prepend(
+                onKey: "OTHER_LDFLAGS",
+                exportedSymbolsLists.flatMap { filePath in
+                    return ["-exported_symbols_list", try filePathResolver.resolve(filePath).string.quoted]
+                }
+            )
+        }
```

Comment:
```suggestion
        if let exportedSymbolsList = target.inputs.exportedSymbolsList {
            try buildSettings.prepend(
                onKey: "OTHER_LDFLAGS",
                ["-exported_symbols_list", try filePathResolver.resolve(exportedSymbolsList).string.quoted]
            )
        }
```

Reply:
I am confused why we now only have a [single exported symbol list when multiple are supported](https://github.com/bazelbuild/rules_apple/blob/3bda1c132ac882c81b1148ff16fbbd08716102aa/doc/rules-macos.md#macos_application-exported_symbols_lists)

Reply:
Ohh, wow, I misread that. You had it right before. We should have two in our example as well then. Sorry 😬.

Reply:
Okay I am going to fall back to the final change before these suggestions and push up again